src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -231,12 +250,13 @@ template <size_t SpatialDim, typename Frame, typename DataType>
 struct DetAndInverseSpatialMetricCompute
     : DetAndInverseSpatialMetric<SpatialDim, Frame, DataType>,
       db::ComputeTag {
+  using argument_tags = tmpl::list<SpatialMetric<SpatialDim, Frame, DataType>>;
+  using base = DetAndInverseSpatialMetric<SpatialDim, Frame, DataType>;
```

Comment:
When #3 is resolved, we'll have to make this consistent, having `using base` or not...ok for now